backend/api/Areas/Tools/Controllers/LtsaController.cs

```diff
@@ -0,0 +1,134 @@
+using Microsoft.AspNetCore.Mvc;
+using Pims.Api.Mapping.Converters;
+using Pims.Api.Policies;
+using Pims.Dal.Security;
+using Pims.Ltsa;
+using Pims.Ltsa.Extensions;
+using Swashbuckle.AspNetCore.Annotations;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Model = Pims.Ltsa.Models;
+
+namespace Pims.Api.Areas.Tools.Controllers
+{
+    /// <summary>
+    /// LtsaController class, provides endpoints to integrate with Ltsa.
+    /// </summary>
+    [Authorize]
+    [ApiController]
+    [Area("tools")]
+    [ApiVersion("1.0")]
+    [Route("v{version:apiVersion}/[area]/ltsa")]
+    [Route("[area]/ltsa")]
+    public class LtsaController : ControllerBase
+    {
+        #region Variables
+        private readonly ILtsaService _ltsaService;
+        private readonly IMapper _mapper;
+        #endregion
+
+        #region Constructors
+        /// <summary>
+        /// Creates a new instance of a LtsaController class.
+        /// </summary>
+        /// <param name="ltsaService"></param>
+        /// <param name="mapper"></param>
+        public LtsaController(ILtsaService ltsaService, IMapper mapper)
+        {
+            _ltsaService = ltsaService;
+            _mapper = mapper;
+        }
+        #endregion
+
+        #region Endpoints
+        /// <summary>
+        /// Make a request to Ltsa for title summaries that match the specified `search`.
+        /// </summary>
+        /// <param name="pid">the parcel identifier to search for</param>
+        /// <returns>An array of title summary matches.</returns>
+        [HttpGet("titleSummaries")]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(Model.TitleSummariesResponse), 200)]
+        [ProducesResponseType(typeof(Pims.Api.Models.ErrorResponseModel), 400)]
+        [SwaggerOperation(Tags = new[] { "tools-ltsa" })]
+        [HasPermission(Permissions.PropertyEdit)]
+        public async Task<IActionResult> FindTitleSummariesAsync(string pid)
+        {
+            var result = await _ltsaService.GetTitleSummariesAsync(ParcelConverter.ConvertPID(pid));
+            return new JsonResult(result);
+        }
+
+        /// <summary>
+        /// Post a new order using default parameters and the passed in titleNumber.
+        /// </summary>
+        /// <param name="titleNumber">the title number to create the order for</param>
+        /// <param name="landTitleDistrictCode">the land title district code</param>
+        /// <returns>The order created within LTSA</returns>
+        [HttpPost("titleOrder")]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(Model.OrderWrapper<Model.TitleOrder>), 200)]
+        [ProducesResponseType(typeof(Pims.Api.Models.ErrorResponseModel), 400)]
+        [SwaggerOperation(Tags = new[] { "tools-ltsa" })]
+        [HasPermission(Permissions.PropertyEdit)]
+        public async Task<IActionResult> PostTitleOrderAsync(string titleNumber, string landTitleDistrictCode)
```

Comment:
Seems odd to call it a Post since the Http method would already infer this.  Why not `SendTitleOrderAsync`?  Again, just a naming preference that isn't required.